docs/ReleaseNotes.md

```diff
@@ -1,6 +1,45 @@
 # Acuminator Release Notes
 This document provides information about fixes, enhancements, and key features that are available in Acuminator.
 
+## Acuminator 1.6.1
+Acuminator 1.6.1 includes the diagnostics, code fixes, and enhancements described in this section, as well as the features that have been implemented in previous versions.
+
+### New Diagnostics and Code Fixes
+In this version, the diagnostic and a code fix have been added for the following issue.
+
+| Code                            | Issue Description                                                                                           | Type    | Diagnostics | Code Fix    | 
+| ------------------------------- | ----------------------------------------------------------------------------------------------------------- | ------- | ----------- | ----------- |
+| [PX1013](diagnostics/PX1013.md) | The action handler that initiates a background operation or is executed by a background operation must return `IEnumerable`. | Error | Available   | Available   |
+
+### Enhancements and Bug Fixes in the Code Map
+Acuminator 1.6.1 includes the following enhancements in the code map:
+
+ - In the text editor of Visual Studio, the symbol to which the code map navigates is now highlighted.
+ - The tooltip for a data view in the code map now keeps the BQL formatting of the source code.
+ - The namespace prefixes have been removed from the tooltips in the code map.
+ - The categories without child nodes are no longer displayed in the code map.
+ - The code map now displays the methods of the base graph that have been overridden through the `PXOverride` attribute.
+ - The performance of the code map has been improved.
+
+ The following bugs have been fixed:
+
+ - The code map didn't support graphs with complex declarations.
+ - The code map could crash due to an unhandled exception from Roslyn.
+ - The code map performed calculations in hidden mode.
+
+### Other Fixed Bugs
+In this version of Acuminator, the following bugs have been fixed:
+
+ - The PX1018 error was displayed for data views defined in the fluent BQL style.
+ - Fluent BQL joins were not supported by the Acuminator colorizer.
+ - Acuminator didn't recognize graphs if they were used as generic type parameters. <br/>**Fix description**: Graphs as generic type parameters are now supported. Multiple diagnostics (such as PX1000, PX1001, PX1003, and PX1005) have been refactored. The PX1005 diagnostic now supports graph extensions.
+
+### Other Enhancements
+Acuminator 1.6.1 also includes the following enhancements:
+
+ - Acuminator now supports Visual Studio 2019. 
+ - Acuminator now includes an option to turn off the suppression mechanism for the Acuminator diagnostics. If you set the value of **Tools > Options > Acuminator > Code Analysis > Suppress selected diagnostics** to _False_, Acuminator ignores the suppression file and displays all diagnostics it has found.
```

Comment:
@k-s-popova the suppression mechanism currently is a feature used only in Acumatica. Maybe we should mention this fact, but I'm not sure. 
@dropsonic, @metallmonster   what do you think?

Reply:
@k-s-popova @SENya1990 let's mention that this is intended for internal use

Reply:
I agree

Reply:
done